scripts/build_lilypond.sh

```diff
@@ -42,8 +53,9 @@ mkdir -p "$LILYPOND_BUILD"
     GHOSTSCRIPT="$GHOSTSCRIPT_INSTALL/bin/gs" \
     GUILE="$GUILE_INSTALL/bin/guile" GUILE_CONFIG="$GUILE_INSTALL/bin/guile-config" \
     PYTHON="$PYTHON_INSTALL/bin/python" PYTHON_CONFIG="$PYTHON_INSTALL/bin/python-config" \
+
```

Comment:
I don't think there should be a blank line between the variables and the call to `configure`

Reply:
Sorry, you're quite correct. That was quite unintentional; I'll fix it right away.